wit/wit.go

```diff
@@ -47,9 +47,11 @@ func (r *RemoteWITServiceCaller) UpdateWITUser(ctx context.Context, updatePayloa
 	}
 
 	remoteWITService, err := CreateSecureRemoteClientAsServiceAccount(ctx, witURL)
+
```

Comment:
This is the WIT service I was talking about above.

Reply:
Understood, let's address this in #536 - I'm happy to work on this issue if you like.